ros_support_interfaces/uc2_interfaces/README.md

```diff
@@ -0,0 +1,7 @@
+# Current status
+This repository stores message, service, action definitions for all the moonshot runs prepared until 31/07/2023, namely `RUN 2` and `RUN 3`.
```

Comment:
What are moonshot runs?

Reply:
No idea, it is also coming from @m-morelli